packages/unmock-server/src/server.ts

```diff
@@ -0,0 +1,125 @@
+import debug from "debug";
+import express = require("express");
+import * as fs from "fs";
+// @ts-ignore
+import helmet = require("helmet");
+import * as http from "http";
+import * as https from "https";
+// import * as os from "os";
+
+import {
+  ISerializedRequest,
+  ISerializedResponse,
+  transform,
+} from "unmock-core";
+import { createUnmockAlgo } from "./unmock";
+
+const httpPort = 8000;
+const httpsPort = 8443;
```

Comment:
This is linked to the port defined in `proxy.ts`; could benefit from a shared definition to avoid potential breakage.

Reply:
Added `constants.ts`.